packages/cli/src/test/integration/integration_test.ts

```diff
@@ -54,8 +55,12 @@ async function serve(dirToServe: string, options: ServerOptions = {}) {
   const address = startResult.server.address();
   if (typeof address === 'string') {
     return `http://${address}`;
-  } else {
+  } else if (!isNull(address)) {
     return `http://${address.address}:${address.port}`;
+  } else {
+    // How you gonna serve without an address?  How is that even a server?  If a
+    // server can't respond to requests, is it *reall* a server?
```

Comment:
Sarcasm is spelled with a `y` at the end here? I mean, "reallY".
```suggestion
    // server can't respond to requests, is it *really* a server?
```